pylmod/gradebook.py

```diff
@@ -155,7 +156,8 @@ def set_grade(
             'assignmentId': assignment_id,
             'mode': 2,
             'comment': 'from MITx {0}'.format(time.ctime(time.time())),
-            'numericGradeValue': str(grade_value)
+            'numericGradeValue': str(grade_value),
+            'isGradeApproved': approve_grades
```

Comment:
This should be `json.dumps(approve_grades)` everywhere it gets passed to the service, as is done in https://github.com/mitodl/PyLmod/blob/cadcf4b7b5ba120e93c7722026a333b984e66627/pylmod/gradebook.py#L70 because python likes uppercase string values of true although it does appear to work with the service anyway, we should try to match their API doc as closely as possible.